data-plane/profiler/resources/config-kafka-broker-webclient.properties

```
@@ -13,5 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+# idleTimeoutUnit is optional and defaults to SECONDS if not specified.
 
 idleTimeout=10000
+idleTimeoutUnit=MILLISECONDS
```

```suggestion
idleTimeoutUnit=MILLISECONDS

```
Can you add an empty newline to make the linter happy?